theme_avantgarde/views/customizations.xml

```diff
@@ -1,60 +1,155 @@
     </xpath>
 </template>
 
-<!-- ======== BANNER ======== -->
- <template id="s_banner" inherit_id="website.s_banner" name="Avantgarde s_banner">
-    <xpath expr="//a[hasclass('btn')]" position="attributes">
-        <attribute name="class" add="btn-lg" separator=" "/>
+<!-- ======== PICTURE ======== -->
+ <template id="s_picture" inherit_id="website.s_picture" name="Avantgarde s_PICTUREcover">
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="pt144 pb128" remove="pt48 pb24" separator=" "/>
+    </xpath>
+    <xpath expr="//h2" position="attributes">
+        <attribute name="style" add="text-align: left;" separator=" "/>
+    </xpath>
+    <xpath expr="//font" position="attributes">
+        <attribute name="class" add="text-break" separator=" "/>
+        <attribute name="style" add="font-weight: bolder;" separator=" "/>
+    </xpath>
+    <xpath expr="//font" position="replace" mode="inner">
+        Design Methodology
+    </xpath>
+    <xpath expr="//p" position="attributes">
+        <attribute name="style" add="text-align: left;" separator=" "/>
+    </xpath>
+    <xpath expr="//p" position="replace" mode="inner">
+        Our design methodology is rooted in our core values.<br/>
+        We combine a wide spectrum of technologies with a good portion of creativity.
+    </xpath>
+    <xpath expr="//img" position="attributes">
+        <attribute name="class" add="img-fluid" remove="img-thumbnail" separator=" "/>
+        <attribute name="src">/web_editor/image_shape/website.s_picture_default_image/web_editor/line/oval_2_line.svg?c2=o-color-2</attribute>
+        <attribute name="data-shape">web_editor/line/oval_2_line</attribute>
+        <attribute name="data-original-mimetype">image/jpeg</attribute>
+        <attribute name="data-file-name">bg_image_14.svg</attribute>
+        <attribute name="data-shape-colors">;o-color-2;;;</attribute>
+    </xpath>
+    <xpath expr="//figcaption" position="attributes">
+        <attribute name="style" add="text-align: right;" separator=" "/>
+    </xpath>
+    <xpath expr="//figcaption" position="replace" mode="inner">
+        We carry out an interdisciplinary design process.
+    </xpath>
+</template>
+
+<!-- ======== COLUMNS ======== -->
+ <template id="s_three_columns" inherit_id="website.s_three_columns" name="Avantgarde s_three_columns">
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="pt120 pb240 o_full_screen_height" remove="pt32 pb32" separator=" "/>
+        <attribute name="data-oe-shape-data">{"shape":"web_editor/Wavy/20","flip":[]}</attribute>
+    </xpath>
+    <xpath expr="//div[hasclass('container')]" position="before">
+        <div class="o_we_shape o_web_editor_Wavy_20"/>
+    </xpath>
+    <xpath expr="//div[hasclass('row')]" position="attributes">
+        <attribute name="class" add="m-1 m-md-0" separator=" "/>
+    </xpath>
+    <xpath expr="//div[hasclass('row')]/div/div" position="attributes">
```

Comment:
`<xpath expr="//*[hasclass('card')]" position="attributes">` seems better.
Same for the 2 others columns.

Reply:
Indeed, it's a better syntax. Done 👍